docs/support.md

```diff
@@ -22,6 +22,6 @@ Many developers and users idle on Freenode.net's IRC network in **[#reactjs on f
 
 ## Twitter
 
-For the latest news about React, [follow **@reactjs** on Twitter](https://twitter.com/reactjs). In addition, you can use the [**#reactjs** hashtag](https://twitter.com/search?q=%23reactjs) to keep up with the latest React news.
+For the latest news about React, [follow **@reactjs** on Twitter](https://twitter.com/reactjs) or use the [**#reactjs** hashtag](https://twitter.com/search?q=%23reactjs).
```

Comment:
Overall I think this is a good change, cleans up the verbiage.  Maybe "and" instead of "or"?  I'm not super familiar with twitter, but I think the two are slightly different operation and the twitter user would want to do both?  @zpao or @spicyj may have thoughts.